darknet_ros/test/ObjectDetection.cpp

```diff
@@ -175,8 +176,9 @@ TEST(ObjectDetection, DetectANYmal) {
   }
 
   ASSERT_TRUE(detectedPerson);
-  EXPECT_LT(centerErrorPersonX, 30);
-  EXPECT_LT(centerErrorPersonY, 30);
+  // TODO: accuracy is still bad but not unacceptable (see images)
+  EXPECT_LT(centerErrorPersonX, 260);
```

Comment:
really very different than what this test was expecting, please confirm this is okay. Image output looked fine to me (see previous PR)